doc/source/ray-overview/learn-more.md

```diff
@@ -1,8 +1,25 @@
 # Learn More
 
-Here are some talks, papers, and press coverage involving Ray and its libraries.
+Here are some general rules of thumb, talks, papers, and press coverage involving Ray and its libraries.
 Please raise an issue if any of the below links are broken, or if you'd like to add your own talk!
 
+## Latency numbers every Ray programmer should know
+
+| Stat | Value |
+| --- | --- | 
+| Typical Ray cluster startup time | ~6 minutes |
```

Comment:
Should we mention what it means by "typical"? It sounds a bit ambiguous to me (how many nodes? what's the cloud? and etc.)